server/routes/administration/room-locations/room-locations.controller.js

```diff
@@ -0,0 +1,240 @@
+(function() {
+  'use strict';
+  const _ = require('lodash');
+  const { validate } = require('validate.js');
+  const { courtroomsDAO } = require('../../../objects/administration');
+  const validator = require('../../../config/validation/create-courtroom');
+  const { replaceAllObjKeys } = require('../../../lib/mod-utils');
+
+  module.exports.getRooms = function(app) {
+    return (req, res) => {
+      return res.redirect(app.namedRoutes.build('administration.room-locations.location.get', {
+        locationCode: req.session.authentication.owner,
```

Comment:
this needs to use a different value. I just spoke with ben and the owner will be the wrong value to use. ping me to pass you the correct ways 😄